mod_cluster/extension/src/main/java/org/wildfly/extension/mod_cluster/ContainerEventHandlerAdapterBuilder.java

```diff
@@ -1,8 +1,9 @@
 package org.wildfly.extension.mod_cluster;
 
+import org.jboss.as.controller.OperationContext;
 import org.jboss.msc.service.ServiceBuilder;
 import org.jboss.msc.service.ServiceTarget;
 
 public interface ContainerEventHandlerAdapterBuilder {
-    ServiceBuilder<?> build(ServiceTarget target, String connector, int statusInterval);
+    ServiceBuilder<?> build(ServiceTarget target, OperationContext context, String connector, int statusInterval);
```

Comment:
I'm just looking at your new CapabilityServiceBuilder now.  I suspect this might allow me to simplify the CapabilityServiceBuilder from wildfly-clustering-common to something like:
```
interface CapabilityBuilder<T> extends Builder<T> {
    @Override
    CapabilityServiceBuilder<T> build(CapabilityServiceTarget target);
}
```
My initial point still stands though.  ContainerEventHandlerAdapterBuilder.build(...) should accept a CapabilityServiceSupport rather than an OperationContext.